src/components/List/List.js

```diff
@@ -1,3 +1,37 @@
 import React from 'react'
 import PropTypes from 'prop-types'
```

Comment:
Make sure to include PropTypes for your Component since you have imported them here.  You can have some basic PropTypes below your component like so:

```js
List.propTypes = { 
  // optional
  isOrdered: PropTypes.bool,
  listItems: PropTypes.array
}
```

Full documentation: https://reactjs.org/docs/typechecking-with-proptypes.html